### ISSUE TITLE
Add variables to collection subfolders

### ISSUE THREAD

**Issue by yaugenka** [a85/POSTMan-Chrome-Extension#11025]
I'm using the following collections structure
Project1 (folder)
--User enpoints (subfolder)
----get (request)
----post (request)
----etc

The endpoint groups (subfolders) may have there own versions numbers. But there is no way to define the version via a subfolder variable. It would be nice to be able to define subfolder variables.

**Comment by shubhbhargav** [comment]
@yaugenka Can you please share an example / screenshot of the collection to help us understand the endpoint groups and their version association?
Do these versions change when someone tries to send a request / run a collection?

**Comment by yaugenka** [comment]
For example I have the following requests
Collections:
project (folder)
--users (subfolder)
----{{project}}/v1/users/4
----{{project}}/v1/users/signin
--tickets (subfolder)
----{{project}}/v4/tickets
----{{project}}/v4/tickets/3

It it was possible to define variables inside subfolders I could have written the requests like this
version=v1
{{project}}/{{version}}/users/4
or like this
base=v1/users
{{project}}/{{base}}/4

**Comment by shubhbhargav** [comment]
Thanks for the example @yaugenka. I am assuming that the version value will differ because of users / tickets APIs / requests going through a different development lifecycle. Is that assumption correct?

If yes, does it make sense to keep the different variables for them (i.e. users_api_version: v1 and tickets_api_version: v4)?

**Comment by yaugenka** [comment]
Yes, they have different development lifecycles and need different variables, and it would be more convinient to define those variable inside their respective subfolders not to mess up the collection root.

**Comment by shubhbhargav** [comment]
Got it. Thanks for the feedback @yaugenka. As of now, we don't have an immediate plan to support this feature but we will keep it in our backlog and update here once we take it up.
